pkg/kubectl/cmd/util/clientcache.go

```diff
@@ -17,30 +17,36 @@ limitations under the License.
 package util
 
 import (
+	fed_client_1_3 "k8s.io/kubernetes/federation/client/clientset_generated/release_1_3"
```

Comment:
I think the code you referred is out-dated, rigth? I agreed with @jianhuiz yesterday that we would use the internalversion clientset for now.